docker/lr-metrics/Dockerfile

```diff
@@ -16,9 +16,10 @@ ENV PATH=/miniconda/bin/:/miniconda/envs/lr-metrics/bin/:/root/google-cloud-sdk/
 
 # install conda packages
 COPY ./environment.yml /
+RUN conda install -n base conda-libmamba-solver && conda config --set solver libmamba
```

Comment:
I'm not familiar with how mamba helps dependencies, in particular where it should be installed.
But if you look at the environment.yml, it's trying to create an env named `lr-metrics`, and here you're installing mamba into the `base` env. 
Is this usually what people do?

Reply:
This has been reverted, so it's no longer relevant, but here's some info for posterity.
I tried to build this docker on a desktop with 16Gb of memory.  No dice.  If I was lucky it just got OOM'd after a few hours.  So I ran out to microcenter and bought 32Gb of memory.  After leaving it overnight it was still trying to solve the environment.  Added libmamba and the docker built promptly (a few minutes--don't remember exactly) using little memory.  Don't know about the correctness of the environments (but I'd think you'd want the solver in the base environment).